lib/timeline/component/item/BackgroundItem.js

```diff
@@ -91,6 +92,10 @@ BackgroundItem.prototype._appendDomElement = function() {
 }
 
 BackgroundItem.prototype._updateDirtyDomComponents = function() {
+  // update dirty DOM. An item is marked dirty when:
+  // - the item is not yet rendered
+  // - the item's data is changed
+  // - the item is selected/deselected
```

Comment:
That's actually not what I meant. I meant a comment block like at line 132. Oh well, at least you DRY'd it.